docker/test-app/test_app/views.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2023 UW-IT, University of Washington
+# SPDX-License-Identifier: Apache-2.0
+
+from django.http import HttpResponse
+from django.views.generic.base import TemplateView, View
+from django.shortcuts import render
+from django.conf import settings
+from django.core.cache import cache
+from uw_spotseeker import Spotseeker
+import json
+import logging
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+def get_token() -> str:
+    token = cache.get(settings.APP_NAME)
```

Comment:
Can be simplified into one line:
`return cache.get(settings.APP_NAME, 'no token in cache')`